ansible/group_vars/do/main.yml

```diff
@@ -23,9 +23,12 @@ k8s_name: "{{ project_name }}"
 bucket_name: "{{ project_name }}"
 bucket_path: "crawls"
 
+registry_name: "{{ project_name }}"
+
 domain: "browsertrix.cloud"
 subdomain: "{{ project_name }}"
 
+
 configure_kubectl: false
 use_do_registry: true
```

Comment:
Do we want this to default to false by default?

Reply:
It's hard to say without knowing how people are using the ansible playbook. 

I assumed that configuration for kubectl would be something we wouldn't want to touch unless explicitly requested, but given that this playbook configures all of it anyway, perhaps it does make more sense to have it default to true?

Reply:
Er, sorry was referring to `use_do_registry` actually, got moved! Yeah, let's set that to false by default